projects/valtimo/components/src/lib/components/form-io/form-io-currency/currency.formio.ts

```diff
@@ -0,0 +1,24 @@
+import {Injector} from '@angular/core';
+import {FormIoCurrencyComponent} from './currency.component';
+import {FormioCustomComponentInfo, registerCustomFormioComponent} from '../../../modules';
+
+const COMPONENT_OPTIONS: FormioCustomComponentInfo = {
+  type: 'currency-custom',
+  selector: 'valtimo-currency',
+  title: 'Currency custom',
+  group: 'basic',
+  icon: 'dollar',
+  schema: {
+    label: 'Currency component',
+    key: 'currency',
+    hideLabel: false,
+    tableView: true,
+    validate: {
+      required: true,
+    },
+  },
+};
```

Comment:
Try to remove the 'custom' stuff and just keep it at currency, see if that works